### PR TITLE
Draw exiting info on light dances

### PR DESCRIPTION
**Pull request by dylwhich**

### CODE REVIEW

File: main/modes/mode_dance.c [hunk 1]
```diff
@@ -71,6 +71,7 @@ void danceNone(uint32_t tElapsedUs, uint32_t arg, bool reset);
 /*============================================================================
  * Variables
  *==========================================================================*/
+const char ledDancesExitText[] = "Exit: Start + Select";
```
Comment (AEFeinstein) [inline main/modes/mode_dance.c:74]: Unless there's a size issue, the other two modes with reminder text say `"Start + Select to Exit"`

**Review by AEFeinstein — CHANGES_REQUESTED** [review]

File: main/modes/mode_dance.c (hunk 1 above)
Comment (dylwhich) [reply main/modes/mode_dance.c:74]: I think I tried that and it was a bit tight. Let me double check though, I may have tried something slightly different...

Comment (dylwhich) [reply main/modes/mode_dance.c:74]: Oh yeah:
image
With no space:
image

Comment (dylwhich) [reply main/modes/mode_dance.c:74]: Hmm...
image

image

image

**Review by AEFeinstein — APPROVED** [review]

**Pull request merged**